hw/top_earlgrey/data/ip/chip_spi_host_testplan.hjson

```diff
@@ -16,9 +16,81 @@
             - Run with min and max SPI clk frequencies and with single, dual and quad SPI modes.
 
             Verify all SPI host instances in the chip.
+
+            Notes for silicon targets:
+            - The testbench that mimics a physical SPI device should measure the clock to verify that the frequencies for each speed mode is correct.
+            This can be done by checking timing of the beginning and the end of the packet.
             '''
+      features: [
+        "SPIHOST.RATE.STANDARD",
+        "SPIHOST.RATE.DUAL",
+        "SPIHOST.RATE.QUAD",
+      ]
       stage: V2
+      si_stage: SV2
       tests: ["chip_sw_spi_host_tx_rx"]
     }
+    {
+      name: chip_sw_spi_host_pass_through
+      desc: '''Verify that the SPI host can be configured in pass through mode.
+
+            - Connect the SPI host to a serial NOR flash device.
+            - Configure an SPI device and that SPI host in pass through mode.
+            - Use the testbench's SPI host to write a word to the flash through the passthrough SPI device and host in OpenTitan.
+            - Read that word back.
+            - Check that the word read back is the same as the one written.
+            '''
+      features: [
+        "SPIHOST.USECASE.SERIALNORFLASH",
+        "SPIHOST.USECASE.PASSTHROUGH",
+      ]
+      stage: V3
+      si_stage: SV3
+      tests: []
+    }
+    {
+      name: chip_sw_spi_host_configuration
+      desc: ''' Verifiy that the polarity and clock divider configurations work appropriately.
+
+            1. Send a packet with no clock division and no polarity switch.
+            2. Send a packet with no clock division and a polarity switch.
+            3. Send a packet with clock divided by two and a polarity switch.
+
+            The testbench should compare the three packets that are received, the second packet should be inverted from the first and the third should be send half as fast as the second.
```

Comment:
Nit: (Super nitty!) I think you probably want a semicolon or a new sentence before "the second packet".

Reply:
I like this super nitty comment :)